common/web/keyboard-processor/src/keyboards/activeLayout.ts

```diff
@@ -327,6 +326,7 @@ export class ActiveKeyBase {
           // It's a computed property!  Copy the descriptor onto the key's object.
           Object.defineProperty(key, prop, descriptor);
         } else {
+          // @ts-ignore
           key[prop] = dummy[prop];
```

Comment:
What is being ignored here?

Reply:
> Element implicitly has an 'any' type because expression of type 'string' can't be used to index type 'ActiveKeyBase'.
> &nbsp;&nbsp;No index signature with a parameter of type 'string' was found on type 'ActiveKeyBase'. ts(7053)

Attempts to do this with precise typing get... ugly.

Reply:
All good, please just add that as a comment in the code to save angst for future maintainers please :)